controller/linodemachine_controller.go

```diff
@@ -317,30 +320,40 @@ func (r *LinodeMachineReconciler) reconcileCreate(
 		linodeInstance = &linodeInstances[0]
 	case 0:
 		// get the bootstrap data for the Linode instance and set it for create config
-		createConfig, err := r.newCreateConfig(ctx, machineScope, tags, logger)
+		createOpts, err := r.newCreateConfig(ctx, machineScope, tags, logger)
```

Comment:
I believe `boot` defaults to true in the API, do we need to set it to `false` explicitly so we can continue configuring it before booting it.

Reply:
see https://github.com/linode/cluster-api-provider-linode/pull/140/files#diff-84b19020b945417b38ec4b095741150823859a32633c49a650471dc6717cd3b8R62